src/entry/logEntry.ts

```diff
@@ -28,6 +32,11 @@ export type LogEntryRequestData = {
   filters: Filters | undefined;
 };
 
+export type IPRequestData = {
```

Comment:
To be correct it should be:
```suggestion
export type IpRequestData = {
```